community/membership.md

```diff
@@ -271,9 +271,20 @@ A change in membership role may be approved by consensus of the Argo Project lea
 
 ## Appointment of Subproject Leads
 
-Leads serve a unique and critical role in the direction and organization of each subproject. Each subproject will generally have two co-leads, who share responsibility. There is no fixed term for leads and they often serve until they retire or, in rare cases, are replaced by [voting](GOVERNANCE.md). Leads are encouraged to give other qualified members of the community an opportunity to serve. Former leads are welcome to nominate themelves again in the future.
+Leads serve a unique and critical role in the direction and organization of each subproject. Each subproject will generally have two co-leads, who share responsibility. There is no fixed term for leads and they often serve until they retire or, in rare cases, are replaced by [voting](GOVERNANCE.md). Leads are encouraged to give other qualified members of the community an opportunity to serve. Former leads are welcome to nominate themselves again in the future.
 
-When a new lead needs to be chosen for a subproject, the availability of the role must be communicated in advance to the Argo maintainer community. Any qualified maintainer may nominate themself for the role. The final decision for the appointment of a new lead will be made by consensus of existing Argo Project leads and approvers.
+When a new lead needs to be chosen for a subproject, the availability of the role must be announced in advance to the Argo maintainer community. Any qualified maintainer may nominate themselves for the role. The final decision for the appointment of a new lead will be made by consensus of existing Argo Project leads and approvers.
+
+The announcement should be posted in the #argo-maintainers Slack channel by an existing lead. The announcement should follow this model:
```

Comment:
Old PR, but stumbled upon this and thought I might mention that Slack channels can be directly linked to for accessibility.
I couldn't find `#argo-maintainers` (private?) but here's the link to `#argo-contributors` as an example: https://cloud-native.slack.com/archives/C020XM04CUW

Right-click the channel > Copy > Copy link

Reply:
Thanks, the maintainer channel is indeed private and invite-only for current maintainers, so no need for that link :)

Reply:
Can still make a link to private channels! The link just won't work for someone without access.

Also might be good to clarify that it's a private channel since this is a public doc

> so no need for that link

Maybe, though I find links to be helpful either way as it's much faster to click than finding it yourself.